_includes/_code/tapbox-d/main.dart

```diff
@@ -0,0 +1,139 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:meta/meta.dart';
+
+void main() {
+  runApp(new MyApp());
+}
+
+// State is managed by TapboxModel, a subclass of ChangeNotifier.
+// The StatefulWidgets register as listeners.
+// TBD: Adam is going to add ListeningWidget to the framework,
+//      so it won't be needed here.
+// If something changes, TapboxModel notifies its listeners.
+
+//---------------------------- ParentWidget ----------------------------
+
+class ParentWidget extends StatefulWidget {
+  @override
+  ParentWidgetState createState() => new ParentWidgetState();
+}
+
+class ParentWidgetState extends State<ParentWidget> {
+  TapboxModel model = new TapboxModel();
+
+  @override
+  Widget build(BuildContext context) {
+    return new Column(
+      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+      children: [
+        new TapboxD(model: model),
+        new TapboxD(model: model),
+      ],
+    );
+  }
+}
+
+//-------------------------- ListeningWidget ---------------------------
+
+abstract class ListeningWidget extends StatefulWidget {
+  ListeningWidget({Key key, this.listenable}) : super(key: key);
+
+  Listenable listenable;
+
+  Widget build(BuildContext context);
+
+  ListeningWidgetState createState() => new ListeningWidgetState();
+}
+
+class ListeningWidgetState extends State<ListeningWidget> {
+  void initState() {
+    super.initState();
+    config.listenable.addListener(handleChanged);
+  }
+
+  void didUpdateConfig(ListeningWidget oldConfig) {
+    if (config.listenable != oldConfig.listenable) {
+      oldConfig.listenable.removeListener(handleChanged);
+      config.listenable.addListener(handleChanged);
+    }
+  }
+
+  void dispose() {
+    config.listenable.removeListener(handleChanged);
+    super.dispose();
+  }
+
+  void handleChanged() {
+    setState(() {});
+  }
+
+  Widget build(BuildContext context) => config.build(context);
+}
+
+//--------------------------- TapboxModel ----------------------------
+
+class TapboxModel extends ChangeNotifier {
```

Comment:
I've added ValueNotifier to the framework to make this easier.  You can replace `TapboxModel` with `ValueNotifier<bool>`